.github/workflows/ci.yml

```
@@ -97,6 +97,22 @@ jobs:
             }}
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: Upload awx jUnit test reports
+        if: >-
+          matrix.test.name == 'api-test'
```

That check is not actually necessary, I dropped it from other jobs.
```suggestion
          !cancelled()
```